build/.bazelbuilderversion-fips

```diff
@@ -0,0 +1,2 @@
+cockroachdb/bazel-fips:20230303-060247
+
```

Comment:
nit: it'd be nice to have the same format as `.bazelbuilderversion` by either adding a new line there or removing the new line here although we usually trim whitespace after reading it just to be sure.

Reply:
Good pont! Fixed.